public/data/common_names.json

```diff
@@ -0,0 +1,3 @@
+{   "ngp": "North Galactic Pole", 
+    "sgp": "South Galactic Pole"}
+    
```

Comment:
Ok I hate to bring it up again because it is such a minor issue, but just fyi, we're looking for a newline character specifically... I think you may have a space in that line hence the red warning symbol appearing here again. Same in galactic_points.json :)